### PR TITLE
[FEM] add initial pressure constraint

### PR DESCRIPTION
**Pull request by donovaly**
- this is needed for fluids

### CODE REVIEW

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7364">pipeline status</a> for feature branch [PR_7364](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7364). Pipeline [612969268](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/612969268) was triggered at [580c750](https://github.com/FreeCAD/FreeCAD/pull/7364/commits/580c7506626303d0c935251ee870060932f9567c). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Fem/femobjects/constraint_initialpressure.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+__title__ = "FreeCAD FEM constraint initial pressure document object"
+__author__ = "Uwe Stöhr"
+__url__ = "https://www.freecadweb.org"
```
Comment (luzpaz) [inline src/Mod/Fem/femobjects/constraint_initialpressure.py:26]: __url__ = "https://www.freecad.org/"

Comment (donovaly) [reply src/Mod/Fem/femobjects/constraint_initialpressure.py:26]: Hmm, I am reluctant. We can do this change but then _all_ files should be updated. At the moment they are all use freecadweb.org and my PR should only add the constraint, not introduce other changes.

If you like, can you make a PR where you make the change for all files? Please bring this to the developer's list if the others agree.

Comment (luzpaz) [reply src/Mod/Fem/femobjects/constraint_initialpressure.py:26]: No thanks, I'm waiting for after toponaming merge to make big sweeping changes.

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7364">pipeline status</a> for feature branch [PR_7364](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7364). Pipeline [613248709](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/613248709) was triggered at [f04bd9e](https://github.com/FreeCAD/FreeCAD/pull/7364/commits/f04bd9e1266fbea68e3a4d3ebadbe777a9def189). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7364">pipeline status</a> for feature branch [PR_7364](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7364). Pipeline [613254581](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/613254581) was triggered at [1655523](https://github.com/FreeCAD/FreeCAD/pull/7364/commits/16555232f3a8618af6cdcd686cae39f72fd2e69a). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request merged**